toolkit/tools/imagecustomizerapi/README.md

```diff
@@ -0,0 +1,16 @@
+# Mariner Image Customizer API
+
+The Mariner image customizer (imgcustomizer) will be released as a standalone tool and
+will provide strong backwards compatibility guarantees (after the first official
+release).
+This is contrast to the Mariner toolkit's new image config, which isn't officially
+released and doesn't provide any backwards compatibility guarantees.
+
+While currently the new image config and imgcustomizer config are very similar, in the
+future there is the possibility they will diverge.
```

Comment:
Few of us had a idle thought about splitting the existing imager tool in two. What are your thoughts on taking the "customization" parts of the imager tool out, and using this tool as an extra step even for offline buidls? (Or at least the same libraries)
That would couple this tool more closely with the image config.json format, although which direction that coupling happens in isn't clear to me.